src/libs/pulumi-cli.ts

```diff
@@ -1,8 +1,9 @@
+import * as fs from "fs";
 import * as os from 'os';
 import * as path from 'path';
 import * as core from '@actions/core';
-import * as io from '@actions/io';
 import * as tc from '@actions/tool-cache';
+import * as makeDir from 'make-dir';
```

Comment:
`@actions/io` supports `await io.mkdirP('path/to/make');` and can also do `await io.mv('path/to/file', 'path/to/dest');`.